src/paperwork/frontend/mainwindow/__init__.py

```diff
@@ -2474,13 +2485,15 @@ def _clear_label_list(self):
     def _readd_label_widgets(self, labels):
         label_widgets = {}
         self.widgets['labels'].freeze_child_notify()
+        #self.widgets['labels'].set_border_width(20)
```

Comment:
Please do not let dead code going around like that.
Either there is a good reason to keep it, in which case you should add a comment explaining why, or there isn't.